### ISSUE TITLE
Suggestions for ADA compliance improvements

### ISSUE THREAD

**Issue by Longkt** [PressMaximum/customify#336]
One challenge I consistently run into though is that a number of your widgets that use icons for a link (social links, search box, search icon) fail ADA compliance because there is no text in the link and no aria-label in the link element. I have found that an aria-label can be easily added in the following lines, and I wonder if you'd consider adding this to your theme:

```
\customify\inc\customizer\configs\header\social-icons.php

LINE: 394
) . $shape . '" ' . $rel . 'target="' . esc_attr( $target ) . '" href="' . esc_url( $item['url'] ) . '" aria-label="' . $item['title'] . '">';


\customify\inc\customizer\configs\header\search-box.php

LINE: 276

<button type="submit" class="search-submit" aria-label="submit search">



\customify\inc\customizer\configs\header\search-icon.php

LINE: 317
<a class="search-icon" href="#" aria-label="open search tool">

LINE: 334
<button type="submit" class="search-submit" aria-label="submit search">
```
